features/base/conference/components/Conference.js

```diff
@@ -1,8 +1,8 @@
 import React, { Component } from 'react';
 
-import { FilmStrip } from '../../filmStrip';
-import { LargeVideo } from '../../largeVideo';
-import { Toolbar } from '../../toolbar';
+import { FilmStrip } from '../../../filmStrip';
+import { LargeVideo } from '../../../largeVideo';
+import { Toolbar } from '../../../toolbar';
```

Comment:
Yes, I was thinking about it too, but I thought having "features/base/conference" and "features/conference" will be confusing for people. I can suggest renaming "features/conference" to "features/conferenceUI".